src/components/Authorization/Authorization.jsx

```diff
@@ -1,145 +1,237 @@
-import React from "react";
-import './Authorization.css'
-import {useState} from "react";
-import eyeIcon from '../../assets/icons/closeEye.png'
+import React from 'react';
+import './Authorization.scss';
+import { useState } from 'react';
+import eyeIcon from '../../assets/icons/eyeIconClose.svg';
 import imageCard from '../../assets/img/author.png';
-import eyeIconOpen from '../../assets/icons/eye-open-16-Regular.svg'
-import {Header} from "../Header/Header";
-import {CommonButton} from "../CommonButton/CommonButton";
-import PropTypes from "prop-types";
-import {Footer} from "../Footer/Footer";
-import {useAppDispatch} from "../../redux/store";
-import {authentication} from "../../features/auth/authReducer";
-import {Path} from "../../constants/path";
-import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
+import eyeIconOpen from '../../assets/icons/eye-open-16-Regular.svg';
+import { Header } from '../Header/Header';
+import { CommonButton } from '../CommonButton/CommonButton';
+import PropTypes from 'prop-types';
+import { Footer } from '../Footer/Footer';
+import { useAppDispatch } from '../../redux/store';
+import { authentication } from '../../features/auth/authReducer';
+import { Path } from '../../constants/path';
+import { Link } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
-
-export const Authorization = ({primary}) => {
+export const Authorization = ({ primary }) => {
   const [passwordVisible, setPasswordVisible] = useState(false);
-  const [email, setEmail] = useState("");
-  const [password, setPassword] = useState ("");
-  const [emailError, setEmailError] = useState("");
-  const [passwordError, setPasswordError] = useState("");
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [emailError, setEmailError] = useState('');
+  const [passwordError, setPasswordError] = useState('');
   const [rememberMe, setRememberMe] = useState(false);
-  const [showSuccessMessage, setShowSuccessMessage] = useState(false);
-  const dispatch = useAppDispatch()
+  const dispatch = useAppDispatch();
+  const [emailInputText, setEmailInputText] = useState('');
+  const [passwordInputText, setPasswordInputText] = useState('');
+
+  const handleRememberMeChange = (event) => {
+    setRememberMe(event.target.checked);
+  };
 
-  const handleRememberMeChange = (e) => {
-    setRememberMe(e.target.checked);
+  const errorMessages = {
+    required: 'Это поле обязательно для заполнения',
+    invalid: 'Неверный логин или пароль',
   };
 
   const handleSubmit = async (event) => {
     event.preventDefault();
-    setEmailError("");
-    setPasswordError("");
-
+    setEmailError('');
+    setPasswordError('');
+    let isValid = true;
     if (!email) {
```

Comment:
One more variant realisation thils fetch. Use scss style. Like this:
const [passwordValid, setPasswordValid] = useState(true);
const [mailValid, setMailValid] = useState(true);
const handleSubmit = () => {
  ....
  setPasswordValid(!!password);
  setMailValid(!!mail);
  if (mailValid || passwordValid) return
 ...
}

<input className=`input ${passwordValid ? '' : 'invalid'}`